.circleci/config.yml

```diff
@@ -1,3 +1,19 @@
+aliases:
+  # Uses the same regex as release-tags-job
+  release-tags-workflow: &release-tags-workflow
+    when:
+      and:
+        - matches: { pattern: "^[0-9]+\\.[0-9]+\\.[0-9]+(?!.*-SNAPSHOT).*$", value: << pipeline.git.tag >> }
+        - not:
+            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+  # Uses the same regex as release-tags-workflow
+  release-tags-job: &release-tags-job
+    filters:
+      tags:
+        only: /^[0-9]+\.[0-9]+\.[0-9]+(?!.*-SNAPSHOT).*$/
```

Comment:
I don't think we tag snapshots right? I believe snapshots the way we do them in android/iOS is done automatically on every push to the `main` branch.

Reply:
Also, I wonder if we should just copye the existing `release-tags` alias in iOS/Android, where we basically ignore just some specific tags and release everything else... Mostly to be consistent.

Reply:
We don't tag them indeed, but this was intended to be defensive against the potential case when we push a tag (e.g. for reference purposes) that we don't want to release. This matches all strings that start with a semver, with any suffix except "SNAPSHOT". 

I'm okay to make it less strict to make it behave like iOS/Android, but I think it should still be a single regex for consistency with the workflow filter.